docs/UserGuide.md

```diff
@@ -726,6 +726,8 @@ MediBase3 data are saved automatically as a JSON file located at: `[JAR file loc
 {: .alert .alert-warning}
 > :exclamation: **Caution:**
 > 
+> When editing the data file, ensure that the application is not running as it might prevent the changes from being saved correctly.
```

Comment:
Nit, but maybe changing to "might prevent the changes from being reflected" sounds better?